<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>room_window</class>
 <widget class="QDialog" name="room_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1021</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">QScrollBar{
background:white;
  width: 10px;
}
QScrollBar::handle{
background:rgb(180, 180, 180, 150);
}
QScrollBar::handle:hover{
background:rgb(150, 150, 150, 180);
}
QScrollBar::add-page{
background:transparent;
}
QScrollBar::sub-page{
background:transparent; 
}
QScrollBar::sub-line{
background:transparent;
}
QScrollBar::add-line{
background:transparent;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>5</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="title_label">
       <property name="text">
        <string>gobang made by 9527</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="min_btn">
       <property name="minimumSize">
        <size>
         <width>27</width>
         <height>22</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#min_btn{
border-image: url(:/img/min.png);
background-color: transparent;
}
QPushButton#min_btn::hover{
background-image: url(:/img/min_hover.png);
background-color: transparent;
}

QPushButton#min_btn::pressed{
background-image: url(:/img/min_pressed.png);
background-color: transparent;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="close_btn">
       <property name="minimumSize">
        <size>
         <width>27</width>
         <height>22</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border-image: url(:/img/close.png);
background-color: transparent;
}
QPushButton::hover{
background-image: url(:/img/close_hover.png);
background-color: transparent;
}

QPushButton::pressed{
background-image: url(:/img/close_pressed.png);
background-color: transparent;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,1,2,2">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QLabel" name="icon_label">
       <property name="minimumSize">
        <size>
         <width>66</width>
         <height>67</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>66</width>
         <height>67</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/img/AppID8.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="count_label">
       <property name="text">
        <string>total person</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QWidget" name="info_widget" native="true">
       <property name="styleSheet">
        <string notr="true">QWidget#info_widget{
background:rgba(255,255,255,100);
}</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QLabel" name="name_label">
          <property name="text">
           <string>Never More</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="record_label">
          <property name="text">
           <string>胜：5 平：5 负：5 胜率：33%</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="score_label">
          <property name="text">
           <string>积分：10</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="content_widget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#content_widget{
background-color: rgba(255, 255, 255, 100);
}
</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,1">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>5</number>
      </property>
      <item>
       <widget class="QWidget" name="left_widget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>480</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget#left_widget{
background-color: rgb(81, 113, 158);
}</string>
        </property>
        <layout class="QGridLayout" name="gridLayout"/>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="right_widget" native="true">
        <property name="minimumSize">
         <size>
          <width>322</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>322</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,1,0,3,0">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="tab_widget" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QWidget#tab_widget{
border-width: 5px;
border-image: url(:/img/TitleBack.png)5 5 5 5;
}</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <property name="spacing">
               <number>10</number>
              </property>
              <item>
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="players_btn">
                <property name="minimumSize">
                 <size>
                  <width>70</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton{
border-width:5px;
border-image: url(:/img/pushbutton.png) 5 5 5 5;
}

QPushButton::hover{
border-width:5px;
border-image: url(:/img/pushbutton_hover.png) 5 5 5 5;
}

QPushButton::pressed{
border-width:5px;
border-image: url(:/img/pushbutton_pressed.png) 5 5 5 5;
}</string>
                </property>
                <property name="text">
                 <string>players</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="rank_btn">
                <property name="minimumSize">
                 <size>
                  <width>70</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton{
border-width:5px;
border-image: url(:/img/pushbutton.png) 5 5 5 5;
}

QPushButton::hover{
border-width:5px;
border-image: url(:/img/pushbutton_hover.png) 5 5 5 5;
}

QPushButton::pressed{
border-width:5px;
border-image: url(:/img/pushbutton_pressed.png) 5 5 5 5;
}</string>
                </property>
                <property name="text">
                 <string>rank</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_5">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QStackedWidget" name="stackedWidget">
           <property name="currentIndex">
            <number>1</number>
           </property>
           <widget class="QWidget" name="page">
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QListWidget" name="player_list">
               <property name="styleSheet">
                <string notr="true">QListWidget{
border:0px;
}
QListWidget::item{
height:30px;
selection-background-color:lightblue;
}
</string>
               </property>
               <property name="alternatingRowColors">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="page_2">
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <property name="spacing">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="rank_1">
                 <property name="minimumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background:white;</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="pixmap">
                  <pixmap resource="../../../img/img.qrc">:/img/00090.png</pixmap>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="rank_2">
                 <property name="minimumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background:white;</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="pixmap">
                  <pixmap resource="../../../img/img.qrc">:/img/00091.png</pixmap>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="rank_3">
                 <property name="minimumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background:white;</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="pixmap">
                  <pixmap resource="../../../img/img.qrc">:/img/00092.png</pixmap>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="rank_4">
                 <property name="minimumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background:white;</string>
                 </property>
                 <property name="text">
                  <string>4</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="rank_5">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background:white;</string>
                 </property>
                 <property name="text">
                  <string>5</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QTableWidget" name="rank_table">
               <property name="styleSheet">
                <string notr="true">border:0px;
selection-background-color:lightblue;</string>
               </property>
               <property name="editTriggers">
                <set>QAbstractItemView::NoEditTriggers</set>
               </property>
               <property name="selectionBehavior">
                <enum>QAbstractItemView::SelectRows</enum>
               </property>
               <property name="showGrid">
                <bool>false</bool>
               </property>
               <property name="rowCount">
                <number>5</number>
               </property>
               <property name="columnCount">
                <number>2</number>
               </property>
               <attribute name="horizontalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <attribute name="horizontalHeaderDefaultSectionSize">
                <number>100</number>
               </attribute>
               <attribute name="horizontalHeaderStretchLastSection">
                <bool>true</bool>
               </attribute>
               <attribute name="verticalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderDefaultSectionSize">
                <number>40</number>
               </attribute>
               <attribute name="verticalHeaderHighlightSections">
                <bool>true</bool>
               </attribute>
               <row/>
               <row/>
               <row/>
               <row/>
               <row/>
               <column/>
               <column/>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextBrowser" name="textBrowser">
           <property name="styleSheet">
            <string notr="true">border:0px;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="chat_widget" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>41</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QWidget#chat_widget{
border-image: url(:/img/myexpression.bmp);
}</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLineEdit" name="lineEdit"/>
            </item>
            <item>
             <widget class="QPushButton" name="send_btn">
              <property name="minimumSize">
               <size>
                <width>24</width>
                <height>23</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton#send_btn{
	border-image: url(:/img/send.png)0 46 0 0;
}
QPushButton#send_btn::hover{
	border-image: url(:/img/send.png)0 23 0 23;
}
QPushButton#send_btn::pressed{
	border-image: url(:/img/send.png)0 0 0 46;
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../img/img.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>close_btn</sender>
   <signal>clicked()</signal>
   <receiver>room_window</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
